web/packages/agenta-auth-ui/src/auth.css

```
@@ -0,0 +1,399 @@
+    border-radius: 8px;
+    font-weight: 500;
+}
+.auth-redesign .ant-btn-primary:hover {
+    filter: brightness(1.03);
+}
+
+/* Inputs (plain elements — the package renders native inputs) */
+.auth-redesign .auth-input {
+    width: 100%;
+    box-sizing: border-box;
+    border: none;
```

Stylelint (17.14.0)

[error] Duplicate property "border" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)